fsrs/optimizer.py

```diff
@@ -684,4 +684,4 @@ def _validate_review_logs() -> None:
 
     class Optimizer:
         def __init__(self, *args, **kwargs) -> None:
-            raise ImportError("The Optimizer class requires torch be installed.")
+            raise ImportError("If you don't want install torch, use https://github.com/open-spaced-repetition/fsrs-rs-python instead")
```

Comment:
```suggestion
            raise ImportError("The Optimizer class requires torch be installed.")
```

An `ImportError` should let the developer know what caused the error, not suggest that the developer switch to alternative python package. Furthermore, this change is not relevant to implementing `uv` for CI which is what this PR is for.